app/experimenter/experiments/models.py

```diff
@@ -545,7 +544,7 @@ class ExperimentVariant(models.Model):
     is_control = models.BooleanField(default=False)
     description = models.TextField(default="")
     ratio = models.PositiveIntegerField(default=1)
-    value = JSONField(blank=True, null=True)
+    value = models.TextField(max_length=10240, blank=False, null=True)
```

Comment:
Sorry I forgot to mention, you don't need to specify the max_length here, the only thing it does is make the text field in the admin comically large 🐶 just take that out and regenerate the migration and we're good to go.